fastlane/Fastfile

```diff
@@ -465,6 +465,28 @@ platform :ios do
     end
   end
 
+  desc "Build and deploy PurchaseTesterSwiftUI"
+  lane :deploy_purchase_tester do
+    match(verbose: true, readonly: true)
+    increment_build_number(
+      build_number: latest_testflight_build_number + 1,
```

Comment:
I'm using the CircleCI build number here, but then, this is probably fine as well 👍

Reply:
I think the latest Xcode versions were supposed to auto-bump build numbers? 
But then again, that might be only if you use Xcode, not if you go through command line

Reply:
Yeah and I use that for my own app now, but it wasn't working without bumping the build number manually like this. So they probably built that feature into the UI only (like when Swift shipped 😂)